src/components/TresScene.ts

```diff
@@ -104,6 +92,22 @@ export const TresScene = defineComponent<TresSceneProps>({
 
     const { activeCamera, pushCamera, clearCameras } = useCamera()
 
+    function setCamera() {
+      const camera = scene.children.find((child: any) => child.isCamera)
```

Comment:
```suggestion
      scene.children.find(camera => (camera as Camera).isCamera)
```

I'm not sure if a camera must always be the direct child of a scene. If not, [traverse](https://threejs.org/docs/#api/en/core/Object3D.traverse) should be used here.